app/router.js

```diff
@@ -6,6 +6,11 @@ var Router = Ember.Router.extend({
 });
 
 Router.map(function() {
+  this.route('admin', {path:'admin'}, function() {
+    this.route('create', {path: 'new'});
```

Comment:
I would recommend putting the `create`, `show` and `update` routes inside of a `menu_items` route to group them together.